app/ui/shared/widgets/list.jsx

```diff
@@ -40,17 +40,22 @@ class List extends Component {
 
   render() {
     return (
-      <ul {...this.props}
+      <ul {...omit(this.props, Object.keys(OmittedContainerProps))}
         className={`widget-list ${LIST_STYLE} ${this.props.className || ''}`}>
-        {this.props.children}
+        {this.props.reversed ? [...this.props.children].reverse() : this.props.children}
```

Comment:
If we only have one child, `this.props.children` is not an array/iterable, so `...this.props.children` would throw `children is not iterable`, the reason for `React.Children.toArray(this.props.children)` utility